examples/counterexamples/README.md

```diff
@@ -0,0 +1,8 @@
+Every CSDL file in this folder represents a test case where schema validation that is meant to fail at a certain line and column because a certain XML schema rule is violated.
```

Comment:
```suggestion
Every CSDL XML file in this folder represents a test case where XML schema validation is meant to fail at a certain line and column because a certain XML schema rule is violated.
```